docs/topics/mta-web-skipping-assessment-of-apps.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-skipping-assessment-of-apps_{context}"]
+= Skipping assessment of applications
+
+By default, the Assessment procedure is required for proceeding to the next step, Reviewing. However, you might want to skip Assessment and use your own questionnaire instead. In that case, use the option described below.
```

Comment:
Anyway we can put `Reviewing` in bold or something? It's a little confusing the way it's written right now.

Possibly something like
>By default, the Assessment procedure is required prior to Reviewing the application

I don't know, maybe it's fine as is if we just explicitly call out that it's an explicit action called `Reviewing`.

Reply:
I've put it in bold and changed to *Review*. This is the name of the next step.

Reply:
Rephrased